Jenkinsfile

```diff
@@ -1261,45 +1197,10 @@ pipeline {
                 }
             }
         } // stage('Test Hardware')
-        stage('Test Report') {
-            parallel {
-                stage('Bullseye Report on EL 8') {
-                    when {
-                        beforeAgent true
-                        expression { !skipStage() }
-                    }
-                    agent {
-                        dockerfile {
-                            filename 'utils/docker/Dockerfile.el.8'
-                            label 'docker_runner'
-                            additionalBuildArgs dockerBuildArgs(repo_type: 'stable') +
-                                " -t ${sanitized_JOB_NAME}-el8 " +
-                                ' --build-arg BULLSEYE=' + env.BULLSEYE +
-                                ' --build-arg REPOS="' + prRepos() + '"'
-                        }
-                    }
-                    steps {
-                        // The coverage_healthy is primarily set here
-                        // while the code coverage feature is being implemented.
-                        job_step_update(
-                            cloverReportPublish(
-                                coverage_stashes: ['el8-covc-unit-cov',
-                                                   'func-vm-cov',
-                                                   'func-hw-medium-cov',
-                                                   'func-hw-large-cov'],
-                                coverage_healthy: [methodCoverage: 0,
-                                                   conditionalCoverage: 0,
-                                                   statementCoverage: 0],
-                                ignore_failure: true))
-                    }
-                    post {
-                        cleanup {
-                            job_status_update()
-                        }
-                    }
-                } // stage('Bullseye Report on EL 8')
-            } // parallel
-        } // stage ('Test Report')
+        // stage('Test Report') {
+        //    parallel {
+        //    } // parallel
+        //} // stage ('Test Report')
```

Comment:
Should we just remove this?

Reply:
Probably in the future.   Comments should not be taking up space in the Jenkinsfile limit.